doc-content/enterprise-only/openshift/operator-deploy-basic-proc.adoc

```diff
@@ -37,7 +37,7 @@ endif::DM[]
 +
 You cannot remove any service from the pod or add any new service to the pod. If you want to use another version of a service or to modify the configuration in any other way, deploy a new server image to replace the old one. You can use any container-based integration workflows to manage the pods.
 +
-When configuring this environment, in the *KIE Servers* tab you must customize the {KIE_SERVER} and click the *Set immutable server configuration* button. For instructions about configuring the {KIE_SERVER}, see xref:operator-deploy-kieserver-proc_{context}[]. 
+When configuring this environment, in the *KIE Servers* tab you must customize the {KIE_SERVER} and either click the *Set immutable server configuration* button or set the `KIE_SERVER_CONTAINER_DEPLOYMENT` environment variable. For instructions about configuring the {KIE_SERVER}, see xref:operator-deploy-kieserver-proc_{context}[]. 
```

Comment:
Suggestion - adopt UI style for buttons from IBMSG 212p:

`s/click the *Set Immutable server configuration* button or/click *Set Immutable server configuration* or`